strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/AuthenticatorsScanner.java

```diff
@@ -70,26 +70,17 @@ public void scanAndReloadRegistry()
     private List<Authenticator> getAuthenticators(ClassLoader currentClassLoader,
                                                   ApplicationContext applicationContext)
     {
-        final List<String> authenticatorsClasses = applicationContext.getBean("authenticators", List.class);
+        final List<Object> authenticatorsClasses = applicationContext.getBean("authenticators", List.class);
         final List<Authenticator> authenticators = new ArrayList<>();
-        for (final String auth : authenticatorsClasses)
+        for (final Object authenticator : authenticatorsClasses)
```

Comment:
@fuss86 pleace note this changes
see [SB-835](https://youtrack.carlspring.org/issue/SB-835) for details